.github/workflows/build.yml

```diff
@@ -25,27 +30,13 @@ jobs:
                     php-version: ${{ matrix.php }}
                     tools: ${{ matrix.tools }}"
                     coverage: none
-            -   name: Get composer cache directory
-                id: composercache
-                run: echo "::set-output name=dir::$(composer config cache-files-dir)"
 
-            -   name: Cache composer dependencies
-                uses: actions/cache@v2
-                with:
-                    path: ${{ steps.composercache.outputs.dir }}
-                    key: composer-${{ runner.os }}-${{ matrix.php }}-${{ hashFiles('composer.*') }}
-                    restore-keys: |
-                        composer-${{ runner.os }}-${{ matrix.php }}-
-                        composer-${{ runner.os }}-
-                        composer-
-
-            -   name: Set composer root version
-                run: |
-                    source .composer-root-version
-                    echo "COMPOSER_ROOT_VERSION=${COMPOSER_ROOT_VERSION}" >> $GITHUB_ENV
+            # set branch alias to allow install humbug/box that depends on 0.14
+            -   run: composer config extra.branch-alias.dev-master 0.14-dev
```

Comment:
This is to keep current `composer.json` untouched. Though in the future would better to keep this line removed and `composer.json` + `humbug/box` synced. We use this approach in Symplify and Rector and it works well :+1:

Reply:
So do I understand correctly that you set `COMPOSER_ROOT_VERSION=dev-master` for forcing PHP-Scoper version to `dev-master` and then setting the alias to match whichever branch you want it to correspond to, i.e. `0.14-dev` here and as we bump versions we would need to bump this alias?

>Though in the future would better to keep this line removed and composer.json + humbug/box synced

What do you mean by this?

Reply:
I already forgot as it's been so long time ago for my contributions, but something like what you say.

By that I mean now humbug requires 2 different version, but box only one. That leads to collision. Ideally they should requires only one last mayor version.